content/guides/getting-started/component-framework-configuration.md

```diff
@@ -189,51 +153,30 @@ test would result in the props being passed into the page to be undefined.
 
 While you could pass in props directly to the page component in a component
 test, that would leave these server-side methods untested. However, an
-end-to-end test would execute and test a page in its entirety.
+end-to-end test would execute and test a page entirely.
 
 Because of this, we recommend using end-to-end testing over component testing
-for Next.js pages, and component testing for individual components.
+for Next.js pages and component testing for individual components in a Next.js
+app.
 
 ### Vite
```

Comment:
Can you call this header "React with Vite" (and the same for the Webpack header, in both the React and Vue sections) so the headers match the links in the framework table at the top of the page? It felt a little weird for me to click a link called "React with Vite" and then and up in a section titled "Vite."